### ISSUE TITLE
ctrl/t

### ISSUE THREAD

**Issue by allonestring** [processing/p5.js-editor#121]
I've just downloaded, unzipped and started the editor. It shows a nice default screen:
function setup() {

}

function draw() {

}

I know that it's just personal preference but I like the open curly brackets on a new line:
function setup() 
{

}

function draw() 
{

}

When I press ctrl/t they jump back onto the same line as the function (or if/else/for). In Processing this doesn't happen. Do you think that we could choose which formatting we like best? Please?

**Comment by antiboredom** [comment]
does anyone else have thoughts on this? 

I'm happy to add this to the to do list, my only issue is that I'm trying to minimize the quantity of preferences to a minimum.

**Comment by lmccart** [comment]
maybe a solution is to do something where you can actually edit the underlying settings file in raw form, if you have enough knowhow to do this. but it's not something beginners have to worry about in a settings menu?

**Comment by antiboredom** [comment]
yeah - that could work. We could also add an "advanced" settings mode that's hidden by default.

**Comment by allonestring** [comment]
Thanks for considering this change, but I wonder if you need to change the formatting at all. Rather than add an option to an advance menu, just leave this particular bit of formatting as the user typed it. Again this is how it is in the original Processing.
As for beginners, I find that it’s easier for them (well the 10-11 yo beginners that I teach) to have the open curly on a new line so that they can more readily see the extent of the lines within the curly brackets.

**Comment by shiffman** [comment]
Protecting the preferences and keeping it to an absolute minimum is key to making a simple, beginner friendly environment (in my view).  The solution @lmccart suggests is exactly how we handle this sort of thing in Processing.  There's a preferences.txt file that has a zillion things in it, but the actual preferences itself is limited to a single page with only the very most key preferences.
